api/proto/teleport/legacy/types/types.proto

```diff
@@ -4017,7 +4017,10 @@ message LockSpecV2 {
 message LockTarget {
   option (gogoproto.goproto_stringer) = false;
   option (gogoproto.stringer) = false;
-
+  // Device was introduced in Teleport 12 and it's reserved to avoid
+  // breaking changes.
+  reserved 8;
+  reserved "Device";
```

Comment:
```suggestion
```

I don't think this is useful; we shouldn't be adding features to v11 without going through master first anyway, and removing a reservation (as it's happening here, from v11 to master) is technically a breaking change in itself.